pkg/controller/v1alpha2/experiment/util/manifest_util.go

```diff
@@ -27,7 +27,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func getMetricsCollectorManifest(experimentName string, trialName string, jobKind string, namespace string, metricNames []string, mcs *katibv1alpha2.MetricsCollectorSpec) (*bytes.Buffer, error) {
+func GetMetricsCollectorManifest(experimentName string, trialName string, jobKind string, namespace string, metricNames []string, mcs *katibv1alpha2.MetricsCollectorSpec) (*bytes.Buffer, error) {
```

Comment:
can we use `podSelector []string` to replace `jobKind` here since MetricsCollector use it just for getting corresponding pod?
we can add a function like `getRunPodSelector(jobKind string)`

this is just a tip, it's ok for me for this PR, too